### DISCUSSION TITLE
Demo Import - Affiliate 2.0 > Fehlerhaft

### DISCUSSION THREAD

**Discussion by Gekko7le** [affiliatetheme/affiliatetheme#30] [Q&A]
Hallo Zusammen

Kurze Frage. Habe eben das Affiliate Theme 2.0 sowie das AFC Plug auf einer neuen Wordpress Umgebung installiert. 
Gibt es keinen Demo Import mehr bzw. funktioniert dieser nicht mehr ?

Wenn ich unter Design / Themes den Demo Import anstossen, ergibt diese bei allen Positionen ''failed''.  Zum Schluss ist Wordpress dann komplett beschädigt und die DB muss geschlöscht werden.

**Comment by endcoreCL** [comment]
Hey,

der Demo Import funktioniert nicht mehr und wird in der kommenden 2.2 auch vollständig gelöscht. Die ist gerade bei einigen Nutzern installiert zum testen. Sofern wir keine Fehler mehr feststellen wird die Version auch released.
